File: Modul 3/Soal 5/PRAK305-2410817210019-AchmadReihanAlfaiz.py
user_input = int(input())

detik = user_input % 60
menit = (user_input // 60) % 60
jam = (user_input // 3600) % 24
hari = user_input // 86400

if user_input >= 86400:
    print(f"\nOutput: \n{hari} hari {jam:02}:{menit:02}:{detik:02}")
else:
    print(f"\nOutput: \n{jam:02}:{menit:02}:{detik:02}")

File: Modul 3/Soal 5/PRAK305-2410817210019-AchmadReihanAlfaiz.c
#include <stdio.h>

int main()
{
     int userInput;
     scanf("%d", &userInput);

     int detik, menit, jam, hari;
     detik = userInput % 60;
     menit = (userInput / 60) % 60;
     jam = (userInput / 3600) % 24;
     hari = userInput / 86400;

     if (userInput >= 86400)
     {
          printf("\nOutput: \n%d hari %02d:%02d:%02d", hari, jam, menit, detik);
     }
     else
     {
          printf("\nOutput: \n%02d:%02d:%02d", jam, menit, detik);
     }

     return 0;
}